src/actions/index.js

```diff
@@ -1,60 +1,39 @@
-export const REQUEST_LOCATION = 'REQUEST_LOCATION';
 export const RECEIVE_LOCATION = 'RECEIVE_LOCATION';
-export const SELECT_LOCATION = 'SELECT_LOCATION';
-export const INVALIDATE_LOCATION = 'INVALIDATE_LOCATION';
 
-export function addLocation(location, air) {
+export function receiveLocation(location, json) {
+  //dispatching an action to the store
+  console.log(location, json);
   return {
-    type: 'ADD_LOCATION',
-    location,
-    air
-  }
-}
-
-export const selectLocation = location => ({
-  type: SELECT_LOCATION,
-  location
-});
-
-export const invalidateLocation = location => ({
-  type: INVALIDATE_LOCATION,
-  location
-});
-
-export const requestLocation = location => ({
-  type: REQUEST_LOCATION,
-  location
-});
-
-export const receiveLocation = (location, json) => ({
-  type: RECEIVE_LOCATION,
-  location,
-  data: 'test', //json.data.children.map(child => child.data),
-  receivedAt: Date.now()
-});
-
+    type: 'RECEIVE_LOCATION',
+    date: json.datetime,
+    name: location,
+    aqi: json.breezometer_aqi,
+    desc: json.breezometer_description
```

Comment:
go with full word `description` for your variable. we're not hurting for space, readability will be your friend in the future.